JS/Exercise_22/index.js

```diff
@@ -0,0 +1,15 @@
+alert('Enter two numbers to see if the first is greater than the second');
+let firstNumber = prompt('Enter the first number');
+let secondNumber = prompt('Enter the second number');
```

Comment:
As a best practice, convert values to numbers before comparing them as numbers. In this case, the values returning from prompt are strings not numbers. Great job though. Don't lose the momentum! ⏩ 🥇

Reply:
Changes have been made!